app/services/ipam/dhcp.rb

```diff
@@ -2,15 +2,21 @@ module IPAM
   class Dhcp < Base
     delegate :dhcp, :dhcp_proxy, :to => :subnet
     def suggest_ip
-      return unless subnet.dhcp?
+      unless subnet.dhcp?
+        message = N_('Failed to fetch a free IP from proxy %{proxy}: %{message}') % {:message => e, :proxy => "#{dhcp.name} (#{dhcp.url})"}
```

Comment:
Looks like copy-paste got you? wasn't this supposed to be something like
```suggestion
        message = N_("Unable to suggest IP: subnet %s don't have a DHCP proxy associated, a proxy is not in subnets organization or IPAM is not set to DHCP") % subnet.name
```